components/CheckoutTable.js

```diff
@@ -0,0 +1,146 @@
+  const totals = useStore(state => state.totals);
+  const ParcelPrice = useStore(state => state.seller.ParcelPrice);
+  const Taxes = useStore(state => state.seller.Taxes);
+  const LocalPickup = useStore(state => state.buyer.LocalPickup);
+  const {
+    TotalTaxes,
+    TotalPrice,
+    TotalShipping,
+    SubTotalPriceInclShipping,
+    TotalParcels,
+  } = totals;
+
+  return (
+    <>
+      <StyledTable>
+        <tr>
+          <th align="left">product</th>
+          <th align="center">QTY</th>
+          <th align="right">unit</th>
+          <th align="right">sum</th>
+        </tr>
+        {productsInCart.map(product => (
+          <tr key={product.id}>
+            <td align="left">
+              <strong>{product.name}</strong>
+            </td>
+            <td align="center">{product.quantity}</td>
+            <td align="right">
+              {product.WSprice.toLocaleString('de-DE', {
+                style: 'currency',
+                currency: 'EUR',
+              })}
+            </td>
+            <td align="right">
+              {product.sum.toLocaleString('de-DE', {
+                style: 'currency',
+                currency: 'EUR',
+              })}
+            </td>
+          </tr>
+        ))}
+        {LocalPickup || (
+          <tr>
+            <td align="left">
+              <strong>Shipping</strong>
+            </td>
+            <td align="center">{TotalParcels}</td>
+            <td align="right">
+              {ParcelPrice.toLocaleString('de-DE', {
+                style: 'currency',
+                currency: 'EUR',
+              })}
+            </td>
+            <td align="right">
+              {TotalShipping.toLocaleString('de-DE', {
+                style: 'currency',
+                currency: 'EUR',
+              })}
+            </td>
+          </tr>
+        )}
+        <tr>
+          <td empty />
+          <td empty />
+          <th align="right">Subtotal</th>
+          <th align="right">
+            {SubTotalPriceInclShipping.toLocaleString('de-DE', {
+              style: 'currency',
+              currency: 'EUR',
+            })}
+          </th>
+        </tr>
+        <tr>
+          <td empty />
+          <td empty />
+          <th align="right">VAT {Taxes}%</th>
+          <th align="right">
+            {TotalTaxes.toLocaleString('de-DE', {
+              style: 'currency',
+              currency: 'EUR',
+            })}
+          </th>
+        </tr>
+        <tr>
+          <td empty />
+          <td empty />
+          <th align="right">TOTAL</th>
+          <th align="right">
+            {TotalPrice.toLocaleString('de-DE', {
+              style: 'currency',
+              currency: 'EUR',
+            })}
+          </th>
+        </tr>
+      </StyledTable>
+    </>
+  );
+}
+
+const StyledTable = styled.table`
+  font-size: 0.9rem;
+  padding: 1rem 0;
+  color: var(--text-maincolor);
+  width: 100%;
+  border-spacing: 0;
```

Comment:
If the border-spacing is 0, I think you don't need this :)

Reply:
it overwrites the default border-spacing, which is 2px.